### PR TITLE
Update UI for API Rule with `introspection_url` and `introspection_request_headers`

### PR DESCRIPTION
**Pull request by barchw**
**Description**

Changes proposed in this pull request:

- Add `introspection_request_headers` access strategy parameter
- Add `introspection_url ` access strategy parameter

### CODE REVIEW

**Review by videlov — CHANGES_REQUESTED** [review]

File: extensions/gateway/apirules.yaml [hunk 1]
```diff
@@ -229,6 +235,11 @@ data:
                   type: array
                   items:
                     type: string
+                introspection_url:
+                  type: string
+                  pattern: ^(https://|http://).*$
```
[the remark this answers is not in the document]
Comment (barchw) [reply extensions/gateway/apirules.yaml:240]: I would leave that as separate task as I looked into it already and setting up the Badge/Alert for this is not trivial

[the remark this answers is not in the document]
Comment (barchw) [reply extensions/gateway/apirules.yaml:240]: Anyway, this rule won't block the creation of `http` issuer, as the actual problem is caused only by the CRD (checked that)

File: extensions/gateway/apirules.yaml [hunk 2]
```diff
@@ -352,12 +376,16 @@ data:
...
       accessStrategies.required_scope: Required Scope
+      accessStrategies.introspection_url: Introspection URL
+      accessStrategies.introspection_request_headers: Introspection Request Headers
       gateway: Gateway
       host: Host
       inputInfo.gateway: Gateway structure, '{GATEWAY}.{NAMESPACE}.svc.cluster.local'
       inputInfo.path: "Path can contain alphanumeric characters and '/', '.', '*', '?', '!', '-', '(', and ')'."
       inputInfo.jwks_urls: "JWKS URLs must start with 'https://' or 'file://'"
       inputInfo.trusted_issuers: "Trusted Issuers must start with 'https://' or 'file://'"
+      inputInfo.introspection_url: "Optional: URL endpoint used for validating the Bearer token"
```
[the remark this answers is not in the document]
Comment (barchw) [reply extensions/gateway/apirules.yaml:387]: It looked more clear to me with the `Optional:` part, but I can remove it for the sake of having it same as the others

[the remark this answers is not in the document]
Comment (barchw) [reply extensions/gateway/apirules.yaml]: https://github.com/kyma-project/kyma-dashboard/pull/207/commits/4f1b805199bd3db2cd3869e557f9246bf1fe9cc3

**Review by videlov — APPROVED** [review]

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

**Pull request merged**